challenge.js

```diff
@@ -42,6 +44,13 @@
  */
 
 // ANSWER
+<body id ="a">
+  <div id ="d">
+    <p1 id="e"></p>
+  </div>
+</body>
+
+document.getElementbyId("e");
```

Comment:
If E had and `id` attribute set to `e`, then `#e` would be a correct CSS selector. We can select it (though not *only* it) using nothing but tag names and parent/child relationships:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.

1/4